src/single.ts

```diff
@@ -169,7 +204,9 @@ export class RegionRatelimit extends Ratelimit<RegionContext> {
       return {
         success,
         limit: tokens,
-        remaining: Math.max(0, tokens - usedTokensAfterUpdate),
+        rateLimit: rates,
```

Comment:
Do we really need to return given `customLimit` back to users?

Reply:
Not necessarily, just in case. What will be the best approach let me know?

Reply:
A sane user would probably use something like:
```
const CUSTOM_RATE = 500
```
So I don't think they will ever need this.

Reply:
Okay I thought since we are returning the limit in response so we should also add the custom limit as well. And yes both are set by the user so they will be aware of it. But yeah I will revert that